### PR TITLE
Remove Usages of Reserved Word 'class' 

### PR DESCRIPTION
**Pull request by feltnerm**
Same text as #110:

According to the ECMAScript specification class is a reserved word. In ES5.1 it was "future reserved word" so I think most browsers let it go, but unfortunately IE9 actually got this one right and barfs when class is used. In the most recent version of ECMAScript (6) class is definitely a reserved word.

There are a few places in the code where this word is used.

I literally just dove into angular-nvd3 today to evaluate its potential for a project, and discovered this when doing some IE9 testing. I don't feel as if I understand the entire scope of a change like this, and was hoping someone would be willing to chime in. It seems, at first glance, that a change like this could result in breaking changes for end-users.

That being said, I'm willing to put together a pull-request if someone is willing to review. Simply changing the variable to be className (or something else) should fix this issue.

tl;dr - class is a reserved word in ECMAScript. It makes IE9 barf. Can we fix it?

### CODE REVIEW

**Comment by acourtiol** [comment]
:+1:

**Comment by NameFILIP** [comment]
Thank you, @feltnerm!
